Framework/PythonInterface/plugins/algorithms/InelasticEMUauReduction.py

```diff
@@ -0,0 +1,460 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import os
+
+# import debugpy
```

Comment:
I think you probably left this in on purpose. If it makes debugging easier, I'm happy for it to stay in